test/browser-tests/router-get/serviceworkers/definition-order.js

```diff
@@ -0,0 +1,37 @@
+/*
+  Copyright 2016 Google Inc. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+'use strict';
+
+/* eslint-env worker, serviceworker */
+
+importScripts('/sw-toolbox.js');
+
+self.addEventListener('install', function() {
```

Comment:
Optional: since you're doing the same `skipWaiting()` + `clients.claim()` setup in each SW file, you can abstract that code out into a separate `sw-setup.js` file, and then add it to the array passed to `importScripts()`.